runtime/queries/ldif/highlights.scm

```diff
@@ -0,0 +1,19 @@
+(comment) @comment
+
+((distinguishedName
+  (name
+    (name_componet
```

Comment:
Is this a typo of "component"?

Reply:
yes https://github.com/kepet19/tree-sitter-ldif/blob/master/grammar.js#L45